src/View/Includes/_footer.html.twig

```diff
@@ -0,0 +1,35 @@
+<div class="footer-dark">
+    <footer>
+        <div class="container">
+            <div class="row text-center">
+                <div class="col-sm-6 col-md-4 item">
+                    <h3>A propos de nous</h3>
+                    <ul>
+                        <li><a href="#">Accueil</a></li>
+                        <li><a href="#">Nous contacter</a></li>
+                    </ul>
+                </div>
+                <div class="col-sm-6 col-md-4 item">
+                    <h3>Services</h3>
+                    <ul>
+                        <li><a href="#">Cours</a></li>
+                        <li><a href="#">Stage</a></li>
+                    </ul>
+                </div>
+                <div class="col-md-4 col-sm-12 item text">
+                    <h3>Haras du Val de Loire</h3>
+                    <ul>
+                        <li>Adresse : 999 rue de Chemeau 45560 Saint Denis en Val</li>
+                        <li>N° de contact : 02 38 24 75 20</li>
+                        <li>Email : contact@harasduvaldeloire.fr</li>
```

Comment:
warning : writing phone numbers and email addresses flat on a web page makes it easy for bots to pick up the informations and add them to spam lists

Reply:
how am i supposed to do ?